src/commonComponents/checkbox/Checkbox.js

```diff
@@ -0,0 +1,31 @@
+// Library
+import PropTypes from 'prop-types'
+
+// css
+import './checkbox.css'
+
+function Checkbox(props){
+
+    const { label, id } = props;
```

Comment:
In commonComponents make a habit of taking a className as a prop

You can see how we have reused IconWithLabel